### PR TITLE
Document IP whitelist restrictions

### PR DESCRIPTION
**Pull request by benh-gds**
This update adds an explanation of the IP whitelist to limit CLI access
to PaaS. See Pivotal story below for details.

If tenant's IP is not whitelisted, they can't use the PaaS at all, and
this is known to have been a problem for several users, so I have added
warnings in the Quick Setup guide, the Troubleshooting section, and in
the Limitations section.

To test: check that explanation is accurate and easy to understand.
Check that link to whitelist is accurate and confirm we should point
users to this information. In "Troubleshooting", check that the
description of what happens if you try to use the client without being
whitelisted is correct.

Pivotal story:
#131354965

Currently we only allow a limited set of IP addresses to access the CF
endpoints (CF, loggregator, UAA).
This is a temporary way of reducing our attack surface until we are
able to do some more work to secure the system from attack.
Things we could and possibly should document for tenants:
that their deployed applications are public and not whitelisted
that the restrictions will prevent them from running cf commands from
an IP address not on the list
that this is a temporary measure
that we can add IP addresses to the whitelist if required by tenants

### CODE REVIEW

File: docs/getting_started/limitations.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 While the Government PaaS is built using Cloud Foundry technology, we don't support all Cloud Foundry features. This section explains some Cloud Foundry features that are not enabled, as well as some limitations of the beta phase.
 
+## IP whitelisting
```
Comment (bleach) [inline docs/getting_started/limitations.md]: Is IP whitelisting well understood enough terminology? Could we make this clearer by making the title something like "you can only manage apps from certain locations / networks"

I wonder if "restrictions" is a slightly plainer English word that could be a drop in for whitelist in most of this text

File: docs/getting_started/limitations.md [hunk 2]
```diff
@@ -34,7 +42,7 @@ In the meantime, we suggest that you use a [blue-green deployment process](https
 
 During the beta period, there may be occasional brief periods where API access is unavailable during a platform update, causing commands sent from the command line client to fail. 
 
-If you find that a valid command is failing and the error message does not explain the problem, please wait 5 minutes before trying the command again. If the error persists for more than 5 minutes, it is unlikely to be caused by a platform update and you should contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+If you find that a valid command is failing and the error message does not explain the problem, please wait 5 minutes before trying the command again. If the error persists for more than 5 minutes, it is unlikely to be caused by a platform update and you should contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
```
Comment (mtekel) [inline docs/getting_started/limitations.md:45]: the added space looks like a typo...

**Comment by mtekel** [comment]
all good, we can live with some additional spaces, merging

**Comment by benh-gds** [comment]
Don't think the extra space affects the final HTML output anyway.

**Pull request merged**